lib/shelley/src/Cardano/Wallet/Shelley/Network.hs

```diff
@@ -1360,6 +1385,7 @@ instance HasSeverityAnnotation NetworkLayerLog where
         MsgWatcherUpdate{}                 -> Debug
         MsgChainSyncCmd cmd                -> getSeverityAnnotation cmd
         MsgInterpreter{}                   -> Debug
-        MsgQueryTime{}                     -> Info
+        MsgQuery _ msg                     -> getSeverityAnnotation msg
+        MsgQueryTime{}                     -> Debug
```

Comment:
I think these are more useful than Debug. How about either
1. Keeping it at Info
2. Debug for sub 100ms, and otherwise a higher severity

Btw, when testing, I noticed `CARDANO_WALLET_TRACING_MIN_SEVERITY=Debug` doesn't actually seem to work - it is still at Info

Reply:
OK I have changed the severity for slow queries ≥ 100ms to Notice.

Thanks for letting me know about the env var. That's fixed now. It was setting the global minimum severity, but not severities for individual tracers, doh!

Reply:
I also now have a way of logging slow queries before they even finish, but that can go in another PR.